app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,115 @@
+<form class="ui form" {{action 'submit' on='submit' }}>
+  <div class="ui segments">
+    <div class="ui aligned secondary segment">
+      <h3 class="weight-400">{{t 'Registration Information'}}</h3>
+    </div>
+    <div class="ui padded segment">
+      <div class="ui yellow inverted segment">
+        <div class="ui tablet stackable grid">
+          <div class="three wide column" style="font-size:24px;margin:10px 0 0 0">
+            08:48
+          </div>
+          <div class="thirteen wide column">
+            <p>
+              {{t 'Please complete registration within 10:00 minutes.'}}
+              <br>
+              {{t 'After 10:00 minutes, the reservation we\'re holding will be released to others.'}}
+            </p>
+          </div>
+        </div>
+      </div>
+      <h4 class="ui horizontal divider header">
+        <i class="ticket icon"></i>
+        {{t 'Ticket Buyer'}}
+      </h4>
+      <div class="field">
+        <label class="required" for="firstname">{{t 'First Name'}}</label>
+        {{input type='text' name='first_name'}}
+      </div>
+      <div class="field">
+        <label class="required" for="lastname">{{t 'Last Name'}}</label>
+        {{input type='text' name='last_name'}}
+      </div>
+      <div class="field">
+        <label class="required" for="email">{{t 'Email'}}</label>
+        {{input type='text' name='email'}}
+      </div>
+      <h4 class="ui horizontal divider header">
+        <i class="ticket icon"></i>
+        {{t 'Ticket Holder\'s Information'}}
+      </h4>
+      {{#each (range 1 3) as |number|}}
```

Comment:
you didn't get what I am trying to say I guess. I won't want to validate all the firstname altogether. If I skip one of them then it shows warning for all of them which I don't want. I want only first one should be validated if I am pointing at that. Apart from that if I add some variable like here number whose value is 1 & 2 here for the loop then how will I be able to validate it?
![image](https://cloud.githubusercontent.com/assets/12701036/26787485/cd60f69e-4a27-11e7-95e4-fa2c10ccb9a2.png)